src/Components/qual-page/QualitativePage.jsx

```diff
@@ -81,9 +86,10 @@ export default function QualitativePage() {
     
        return ( 
         <div className="background">   
-        <form onSubmit={Submit}>
         <FormControl fullWidth>
-          <InputLabel id='select label' style={{color:'#ff5000'}}>{MenuItems[0]}</InputLabel>
+          <InputLabel id='select label' style={{color:'#ff5000'}}>
```

Comment:
Element IDs should not contain a space; this gives it two IDs: `select` and `label`